Editor/Window/GameLiftRequestAdapter.cs

```diff
@@ -0,0 +1,236 @@
+        private VisualElement _container;
+
+        public GameLiftRequestAdapter(GameLiftPlugin gameLiftConfig)
+        {
+            _gameLiftConfig = gameLiftConfig;
+        }
+
+        internal async Task<bool> CreateAnywhereFleet(string fleetName)
+        {
+            if (_gameLiftConfig.GameLiftWrapper != null)
+            {
+                var success = await CreateCustomLocationIfNotExists(FleetLocation);
+                if (success)
+                {
+                    var fleetId = await CreateFleet(ComputeType.ANYWHERE, FleetLocation,fleetName);
+                    var fleetNameResponse = _gameLiftConfig.CoreApi.PutSetting(SettingsKeys.FleetName, fleetName);
+                    var fleetIdResponse = _gameLiftConfig.CoreApi.PutSetting(SettingsKeys.FleetId, fleetId);
+                    var customLocationNameResponse =
+                        _gameLiftConfig.CoreApi.PutSetting(SettingsKeys.CustomLocationName, FleetLocation);
+                    return fleetNameResponse.Success && customLocationNameResponse.Success && fleetIdResponse.Success;
+                }
+
+                return false;
+            }
+
+            Debug.Log("Error: Missing Account Profile");
+            return false;
+        }
+        
+        private async Task<bool> CreateCustomLocationIfNotExists(string fleetLocation)
+        {
+            try
+            {
+                var listLocationsResponse = await _gameLiftConfig.GameLiftWrapper.ListLocations(new ListLocationsRequest
+                {
+                    Filters = new List<string> { "CUSTOM" }
+                });
+
+                var foundLocation =
+                    listLocationsResponse.Locations.FirstOrDefault(l => l.LocationName.ToString() == fleetLocation);
+
+                if (foundLocation == null)
+                {
+                    var createLocationResponse = await _gameLiftConfig.GameLiftWrapper.CreateLocation(
+                        new CreateLocationRequest()
+                        {
+                            LocationName = fleetLocation
+                        });
+
+                    if (createLocationResponse.HttpStatusCode == HttpStatusCode.OK)
+                    {
+                        Debug.Log($"Created Custom Location {fleetLocation}");
+                    }
+                }
+
+                return true;
+            }
+            catch (Exception ex)
+            {
+                var errorBox = _container.Q<VisualElement>("FleetErrorInfoBox");
+                errorBox.style.display = DisplayStyle.Flex;
+                errorBox.Q<Label>().text = ex.Message;
+                return false;
+            }
+        }
+        
+        private async Task<string> CreateFleet(ComputeType computeType, string fleetLocation, string fleetName)
+        {
+            try
+            {
+                var createFleetRequest = new CreateFleetRequest
+                {
```

Comment:
So we can track usage of the plugin feature, please add a tag to these fleets:
```
{ Key: "CreatedBy", Value: "AmazonGameLiftUnityPlugin" }
```

Reply:
Please also add the following string for the fleet description property:
```
Deployed by the Amazon GameLift Plug-in for Unity.
```

Reply:
Added both tag and new description